cmd/ocm/cluster/create/create.go

```diff
@@ -93,6 +100,9 @@ func run(cmd *cobra.Command, argv []string) error {
 	if len(args.expirationTime) > 0 && args.expirationSeconds != 0 {
 		return fmt.Errorf("at most one of `expiration-time` or `expiration` may be specified")
 	}
+	if args.region == "us-east-1" && args.provider != "aws" {
+		return fmt.Errorf("if specifying a non-aws cloud provider, region must be set")
```

Comment:
Probably not too big of a deal, but this message would be misleading if the user explicitly set the region to "us-east-1".

Reply:
That's true, but I wanted to maintain the current backward compatible aws defaults, *and* I want cobra to display them in help text. I'm open to suggestions on how to better word this, or check for this use case.

Reply:
I had another look at this.. it will only fire if the provider is NOT aws, and `us-east-1` isn't a valid region in gcp. I amended the language to add the word valid if someone actually specifies the combination of `--provider=gcp --region=us-east-1` manually.